addons/website/static/tests/builder/website_builder/image_gallery.test.js

```diff
@@ -15,8 +15,10 @@ import {
 
 defineWebsiteModels();
 
-async function setupWbsiteBuilderWithImageWall() {
-    const builder = await setupWebsiteBuilderWithSnippet("s_images_wall");
+async function setupWbsiteBuilderWithImageWall(loadIframeBundles = false) {
+    const builder = await setupWebsiteBuilderWithSnippet("s_images_wall", {
+        loadIframeBundles,
+    });
```

Comment:
Refereing to this https://github.com/odoo/odoo/pull/240700/changes/bd26d3c1dfbd1cafd19098f04249dfc46a4933bf#r2650215540 this change now is unnecessary.